### ISSUE TITLE
Improve performance for very large tracings

### ISSUE THREAD

**Issue by philippotto** [scalableminds/webknossos#5088]
There are multiple problems regarding very large tracings. Let's focus on the upload-NML usecase for now since this happens every now and then:

- the upload can take quite long and if one has bad luck, it doesn't succeed. also, it's not clear that there is a difference in front-end vs back-end upload.
- if the upload succeeded, opening the tracing can take quite long and sometimes it fails.
- moving in the annotation as the owner creates new update actions which can also be problematic.

We should have a meeting to discuss these problems and find mitigations/solutions for them.

**Comment by philippotto** [comment]
As short-term goals we defined in our meeting from last Thursday:
- [ ] investigate why view-requests for a large tracing can fail
	- are there more timeouts to tackle?
	- was the healthcheck at fault, because it was blocked by fossildb? see https://github.com/scalableminds/webknossos/issues/5005
	- doublecheck autoheal logs
- [ ] for very large nmls (e.g., > 100 MB ?), encourage the user to use the dashboard-import

Mid-term goals:
- consolidate frontend vs. backend upload
	- do we still need both?
	- how can we get the best of both approaches?
- "A website is slowing down your browser" should not happen
	- frontend shouldn't block the main thread for that long

High-level goals:
- Viewing/Editing tracings should scale well
	- Note how agglomerate files also represent a kind of tracing
- Multi-user collaboration with tracings should be possible/streamlined
	- E.g., viewing the tracings of all users for a given dataset
	- E.g., working with multiple users in gdoc-style

**Comment by fm3** [comment]
I did some testing and digging through code and determined that the fossildb health check is not blocked by other requests (at least not in the client. I assume the server doesn’t either, but I created separate issue in https://github.com/scalableminds/fossildb/issues/34

A simple explanation for why large view requests failed was the 75s limit that we thought we had disabled but in fact for standalone tracingstores we hadn’t, compare https://github.com/scalableminds/webknossos/pull/5174
